components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -2215,23 +2216,15 @@ func handleSpecialChar(term string) string {
 	return fmt.Sprintf("(%s)", term)
 }
 
+//filterQueryChange Will return day latest only for last 24 hours
 func filterQueryChange(endTime string, startTime string) ([]string, error) {
 	if len(endTime) == 0 && len(startTime) == 0 {
 		return []string{"day_latest", "daily_latest"}, nil
 	}
 	if len(startTime) == 0 {
 		return []string{"daily_latest"}, nil
 	}
-	eTime, err := time.Parse(layout, endTime)
-	sTime, err := time.Parse(layout, startTime)
-	diff := int(eTime.Sub(sTime).Hours() / 24)
-	if err != nil {
-		return nil, errors.Errorf("cannot parse the time")
-	}
-	if diff == 0 {
-		return []string{"daily_latest"}, nil
-	}
-	return []string{"day_latest", "daily_latest"}, nil
+	return []string{"daily_latest"}, nil
```

Comment:
please validate the unit tests. Tests are failing https://buildkite.com/chef-oss/chef-automate-main-verify/builds/30146#0189bf0a-efbc-4e6b-9956-8cf8a4ba6b2a